src/Microsoft.NetFramework.Analyzers/Core/MvcAttributeSymbols.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.NetFramework.Analyzers
+{
+    public partial class MarkVerbHandlersWithValidateAntiforgeryTokenAnalyzer
```

Comment:
>MarkVerbHandlersWithValidateAntiforgeryTokenAnalyzer [](start = 25, length = 52)

Nit: Naming convention for files containing nested types includes the containing type name, for example here it will be `MarkVerbHandlersWithValidateAntiforgeryTokenAnalyzer.MvcAttributeSymbols.cs`